components/_provisional.scss

```diff
@@ -286,23 +286,14 @@
 		}
 	}
 
-	/* Theme/topic decorative image */
-	&.gc-topic-bg div {
-		background: no-repeat;
-		margin-left: -15px;
-		min-height: 200px;
-		overflow: hidden;
-		width: calc(50vw - 15px);
-	}
-
 	/* Contributors */
 	&.gc-contributors {
 		margin-top: 38px;
 
 		h2, ul {
 			display: inline;
 			font-size: $small-size;
-			margin-top: 0;
+			margin-top: $space-0;
```

Comment:
@duboisp For this one, we should tackle it in a different PR because Institutional landing page doesn't use the GC Contributor component.

Reply:
Then it might more valuable to keep separate the replacement of the `$space-` variable or limit it only to the institutional profile for now if used in more than 2 places.